src/app/api/v1/user/update-comm-option/route.ts

```diff
@@ -19,24 +20,34 @@ import {
 export type EmailUpdateCommTypeOfOptions = "-1" | "0" | "1";
 
 export interface EmailUpdateCommOptionRequest {
-  communicationOption: EmailUpdateCommTypeOfOptions;
+  instantBreachAlerts: EmailUpdateCommTypeOfOptions;
+  monthlyMonitorReport: boolean;
 }
 
 export async function POST(req: NextRequest) {
   const token = await getToken({ req });
 
   if (typeof token?.subscriber?.fxa_uid === "string") {
     try {
-      const { communicationOption }: EmailUpdateCommOptionRequest =
-        await req.json();
+      const {
+        instantBreachAlerts,
+        monthlyMonitorReport,
+      }: EmailUpdateCommOptionRequest = await req.json();
       const subscriber = await getSubscriberByFxaUid(token.subscriber?.fxa_uid);
       if (!subscriber) {
         throw new Error("No subscriber found for current session.");
       }
+      // -1 = Set as null, do not send instant notifications. Newly added in MNTOR-1368
       // 0 = Send breach alerts to the corresponding affected emails.
       // 1 = Send all breach alerts to user's primary email address.
-      const allEmailsToPrimary = Number(communicationOption) === 1 ?? false;
+      const allEmailsToPrimary =
+        Number(instantBreachAlerts) === -1
+          ? null
+          : Number(instantBreachAlerts) === 1
+            ? true
+            : false;
```

Comment:
I find this double-ternary kind of difficult to read... you might consider defining `Number(instantBreachAlerts)` to a short variable once. I suspect the `? true : false` part is probably not necessary since `Number(instantBreachAlerts) === 1` the `===` will always evaluate to `true` or `false`

Reply:
this whole number thing seems silly (re @Vinnl's comment above).. let's just change it to string enums